cypress/locators/simple-color-picker/locators.js

```diff
@@ -1,2 +1,3 @@
 // component preview locators
 export const SIMPLE_COLOR_PICKER_PREVIEW = 'div[data-component="simple-color-picker"] > ul';
+export const EXPERIMENTAL_SIMPLE_COLOR_PICKER = 'fieldset[data-component="simple-color-picker"]';
```

Comment:
Please, check if you need `fieldset` in this locator. If not remove it. As below is enough
```suggestion
export const EXPERIMENTAL_SIMPLE_COLOR_PICKER = '[data-component="simple-color-picker"]';
```

Reply:
fixed

Reply:
fixed